src/wp-includes/script-loader.php

```diff
@@ -214,6 +214,39 @@ function wp_get_script_polyfill( $scripts, $tests ) {
 	return $polyfill;
 }
 
+/**
+ * Registers development scripts that integrate with `@wordpress/scripts`.
+ *
+ * @see https://github.com/WordPress/gutenberg/tree/trunk/packages/scripts#start
+ *
+ * @since 6.0.0
+ *
+ * @param WP_Scripts $scripts WP_Scripts object.
+ */
+function wp_register_development_scripts( $scripts ) {
+	if ( ! defined( 'SCRIPT_DEBUG' ) || ! SCRIPT_DEBUG ) {
+		return;
+	}
+
+	$development_scripts = array(
+		'react-refresh-entry',
+		'react-refresh-runtime',
+	);
+
+	foreach ( $development_scripts as $script_name ) {
+		$assets = include ABSPATH . WPINC . "/js/dist/development/$script_name.php";
+		$scripts->add(
+			'wp-' . $script_name,
```

Comment:
The concatenation operator (`.`) is being used here (which is fine).
But, should the other places in this function have to be updated to use the concatenation operator too (for consistency and readability)?